src/components/TopBar/ProjectPicker.jsx

```diff
@@ -23,22 +26,58 @@ const ProjectPicker = createMenu({
       isUserAuthenticated;
   },
 
-  renderItems({currentProjectKey, projectKeys, onChangeCurrentProject}) {
-    return map(projectKeys, projectKey => (
+  renderItems({
+    currentProjectKey,
+    projects,
+    shouldShowArchivedProjects,
+    onChangeCurrentProject,
+    onToggleViewArchived,
+  }) {
+    const visibleProjects = shouldShowArchivedProjects ?
+      projects :
+      filter(projects, ({isArchived}) => !isArchived);
+    const items = map(visibleProjects, project => (
```

Comment:
Seems like we're still only using `projectKey` in the loop—let's destructure the parameter to reduce duplication?